apps/zk-explorer/components/LinkCard.tsx

```diff
@@ -0,0 +1,22 @@
+import { PropsOf } from '@webb-tools/webb-ui-components/types';
+import { PageUrl } from '../utils/utils';
+import { FC } from 'react';
+import { Card } from '@webb-tools/webb-ui-components';
+
+export type LinkCardProps = PropsOf<'a'> & {
+  href: PageUrl;
+};
+
+export const LinkCard: FC<PropsOf<'a'>> = (props) => {
+  return (
+    <a
```

Comment:
consider using the Link components provided by Next.js [here](https://nextjs.org/docs/pages/api-reference/components/link) instead of <a> tag